src/contract-defs.ts

```diff
@@ -6,7 +6,7 @@ import { Interface } from 'ethers/lib/utils'
 export const getContractDefinition = (name: string, ovm?: boolean): any => {
   const match = glob.sync(
     path.resolve(__dirname, `../artifacts`) +
-      `/**/${name}${ovm ? '.ovm' : ''}.json`
+      `/**/${name}${ovm ? '-ovm' : ''}.json`
```

Comment:
Any test coverage around this?

Reply:
Nope :-( but this is an effect of the latest plugins update.